src/UserService.Validation/Image/AddImageRequestValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User;
+using LT.DigitalOffice.UserService.Validation.Image.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.UserService.Validation.Image
+{
+  public class AddImageRequestValidator : AbstractValidator<AddImageRequest>, IAddImageRequestValidator
+  {
+    private readonly List<string> imageFormats = new()
+    {
+      ".jpg", ".jpeg", ".png", ".bmp", ".gif", ".tga"
+    };
+
+    public AddImageRequestValidator()
+    {
+      RuleFor(x => x.Content)
+        .NotEmpty().WithMessage("Content can't be empty")
+        .Must(content =>
+        {
+          try
+          {
+            var byteString = new Span<byte>(new byte[content.Length]);
+            return Convert.TryFromBase64String(content, byteString, out _);
```

Comment:
Where there is may be throw exception?

Reply:
in content.length maybe
or if content is null

Reply:
why not just use operator compare if?

Reply:
will change this validator to check that content is image in other task